renku/cli/dataset.py

```diff
@@ -883,3 +907,42 @@ def update(short_names, creators, include, exclude, ref, delete, external):
         progress_context=progress_context
     )
     click.secho('OK', fg='green')
+
+
+@dataset.command('mv')
```

Comment:
I wonder if it isn't better to name the command `mv-files`? This would be more consistent with the commands like `rm`, which operates on datasets (but not files).

I tried to rename a dataset by `mv`, but this didn't work. If the command were `mv-files`, it would be clear that I couldn't rename the dataset using mv.

Reply:
We are trying (mostly) to follow git commands names; for example we also have `renku dataset add/unlink` that also work on files. I am not sure if `mv-files` helps here because we normally don't move datasets and can only rename them (renaming is not supported yet BTW).
The main point here is that we use the same `dataset` verb to act on files and datasets. Maybe we should resume the discussion regarding this to separate them (i.e. `renku file` vs `renku dataset`).